docs/sources/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1093,6 +1093,11 @@ Status Codes:
 -   **200** – no error
 -   **500** – server error
 
+> **Note**:
+> When the `remote` parameter references a Dockerfile then the `dockerfile`
+> parameter will have effect and the contents of the Dockerfile are placed
+> into a file called `Dockerfile`.
+
```

Comment:
I don't understand why this note is in returns;if you did this for a specific reason let me know. Otherwise, I'd move this into the parameters section.

The interaction between dockerfile and remote is unclear.  Can they specify either or both?  There seems to be a case where `dockerfile` will not have an effect.  By "git" do you mean an actual file in a git repository? Is there a reason you say git as opposed to filename?

-   **dockerfile** - path within the build context to the Dockerfile.
    For this parameter to have an effect, the caller must also pass a Dockerfile using the `remote` parameter.

-   **remote** – file or HTTP/HTTPS URI build source.  If the caller passes a file, the contents are placed into a file named `Dockerfile` in the location specified by `dockerfile` parameter.